projects/mercury/src/common/utils/__tests__/genericUtils.js

```diff
@@ -87,7 +87,7 @@ describe('isNonEmptyValue', () => {
         values.forEach(v => expect(isNonEmptyValue(v)).toBe(true));
     });
     it('Returns false for the given values', () => {
-        const values = [undefined, null, '', NaN, "", ``];
+        const values = [undefined, null, '', NaN, '', ''];
```

Comment:
this is not valid

Reply:
good point, rolled back the line and disable eslint for the line